source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -54,6 +54,8 @@ Your console output in Visual Studio Code should look like the below. However, t
 
 .. important:: Simulation can also be run outside of VS Code using ``./gradlew simulateJava`` for Java or ``./gradlew simulateNative`` for C++.
 
+.. note:: Some vendors support attaching hardware to your PC and desktop simulation. See :doc:`vendor documentation <wpilib-commands-vscode>` for more information about the command WPILib: Hardware Sim Robot Code.
```

Comment:
Since wpilib-commands-vscode isn't in the same directory as this article, you need to use the full path